docs/components/navigation/index.html

```diff
@@ -35,27 +35,17 @@ <h2 id="left-navigation">Left (Vertical) Navigation</h2>
             L1 Section
             <hx-icon class="toggle-icon" type="angle-down"></hx-icon>
           </hx-disclosure>
-          <hx-reveal id="section-1" open>
+          <hx-reveal class="first-section" id="section-1" open>
             <a href="#">Link 2-1</a>
-            <a href="#">Link 2-2</a>
+            <a href="#" class="hxDisabled">Link 2-2</a>
             <a href="#">Link 2-3</a>
-
-            <hx-disclosure aria-controls="section-1-2">
-              L2 Section
-              <hx-icon class="toggle-icon" type="angle-down"></hx-icon>
-            </hx-disclosure>
-            <hx-reveal id="section-1-2" open>
-              <a href="#">Link 3-1</a>
-              <a href="#">Link 3-2</a>
-              <a href="#">Link 3-3</a>
-            </hx-reveal>
           </hx-reveal>
         </nav>
       </div>
 
       <footer>
         {% code 'html' %}
-          <nav class="hxNav">
+          <nav class="hxNav sub-leftNav">
```

Comment:
Please modify to add a `hx` prefix to any Helix classes we add:

### Please change:

```css
sub-leftNav
```

### To:

```css
hxLeftNavSection
```

Reply:
Changes have been done, Please review